bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+
+The `updateGif` switch can be turned on with a rule triggered by an external zwave PIR sensor or the cameras own motion alarm, the choice and the logic can be created by yourself.
+The feature has two options called preroll and postroll to be aware of.
+When preroll is 0 (the default) the binding will use the RTSP stream to fetch the amount of seconds specified in the postroll config to create the GIF from.
+By changing to a preroll value above 0 the binding will change to using snapshots as the source.
+The time between the snapshots then becomes the polling time of the camera (1 seconds by default) and can be raised if you desire.
+The snapshots are saved to disk and can be used as a feature that is described in the snapshot section above in more detail.
+
+You can request the gif by using this url, or by the path to where the file is stored:
+
+<http://OpenhabIP:ServerPort/ipcamera.gif>
+
+
+.items
+
+```java
+Switch DoorCamCreateGif "Create animated GIF" { channel="ipcamera:DAHUA:DoorCam:updateGif" }
+
+```
+
+.rules
+
+```java
+rule "Create front door camera GIF when front doorbell button pushed"
+when
+    Item FrontDoorbellButton changed to ON
+then
+    //Start creating the GIF
+    DoorCamCreateGif.sendCommand(ON)
+    //Cast a doorbell sound using the Chromecast binding.
+    KitchenHomeHubPlayURI.sendCommand("http://192.168.1.8:8080/static/doorbell.mp3")
+end
+
+rule "Send doorbell GIF via Pushover"
+when
+    Item DoorCamCreateGif changed to OFF
+then
+    sendPushoverMessage(pushoverBuilder("Sending GIF from backyard").withApiKey("dsfhghj6546fghfg").withUser("qwerty54657").withDevice("Phone1").withAttachment("/tmpfs/DoorCam/ipcamera.gif"))
+end
+```
+
+## Group Displays
+
+The full example section shows how to setup a group display.
+
+Some additional things to check to get it working are:
+
++ Currently the poll time of the group must be the same or less than the total time contained in each cameras m3u8 files.
+If you have 3 seconds worth of video segments in the cameras HLS stream, this is the max time you can set to be the Poll time of the group to.
+If your not using HLS and are just using ipcamera.jpg to display the groups picture, then the poll time can be set to a wider range.
++ All cameras should have the same HLS segment size setting. 1 and 2 second long segments have been tested to work.
+
+
+This is still a very new feature and if you have any issues please send some TRACE level log output of when the problem occurs.
+
+## Batch motion detection rules
```

Comment:
I don't know if all the above should be part of this document. It's more like a How To section, which makes this doc even more larger.

Reply:
@kubawolanin Contributed this and helped with the readme a while back. Perhaps he can add it to the forum as a How To article?
Will remove it to make the readme shorter as suggested.